sensor_msgs/JointState[] goals
---
bool success
